### ISSUE TITLE
Error during npm install

### ISSUE THREAD

**Issue by teoSalvetti** [webrtc/apprtc#232]
I have this error when run ```npm install``` on OS X:
```
npm install
npm ERR! Darwin 15.0.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v5.1.0
npm ERR! npm  v3.3.12
npm ERR! code EREADFILE

npm ERR! Error extracting /Users/matteo/.npm/grunt-htmlhint/0.9.9/package.tgz archive: ENOENT: no such file or directory, open '/Users/matteo/.npm/grunt-htmlhint/0.9.9/package.tgz'
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/matteo/Downloads/apprtc-master/npm-debug.log
```

How can I fix it?

**Comment by KaptenJansson** [comment]
Looks like this is the same as https://github.com/npm/npm/issues/10184, Maybe try as the duped bug suggests and install a specific version of grunt-htmlhint? try `npm install grunt-htmlhint@0.9.9`

**Comment by antonmry** [comment]
This worked for me but executing before

```
npm install grunt@0.4.1
```

Thanks!

**Comment by KaptenJansson** [comment]
Fixed in #248
